E2ETests/README.md

```diff
@@ -0,0 +1,88 @@
+# End to End Tests
+
+[Synthetics for Mobile](https://docs.datadoghq.com/mobile_app_testing/) runs E2E test scenarios. [Monitors](https://docs.datadoghq.com/monitors/) assert the proper propagation of data.
+
+
+## CI
+
+CI continuously builds, signs, and uploads a runner application to Synthetics which runs predefined tests daily.
+
+### Build
+
+Before building the application, the `E2ETests/xcconfigs/E2E.local.xcconfig` configuration file must be present and contain the `Mobile - Integration Org` client token and RUM application ID. These values are sensitive and must be securely stored.
+
+```ini
+CLIENT_TOKEN=
+RUM_APPLICATION_ID=
+```
+
+> [!TIP]
+> Files can be base64 encoded and stored in secret variables. To copy the encoded file content, you can do:
+> ```bash
+> cat E2ETests/xcconfigs/E2E.local.xcconfig | base64 | pbcopy
+> ```
+
+### Sign
+
+To sign the runner application, the certificate and provision profile defined in [Synthetics.xcconfig](xcconfigs/Synthetics.xcconfig) and in [exportOptions.plist](exportOptions.plist) needs to be installed on the build machine. These files are sensitive and must be securely stored. Make sure to update both files when updating the certificate and provisioning profile, otherwise signing fails.
+
+> [!NOTE]
+> Certificate & Provisioning Profile could be downloaded using [App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi) instead of stored CI secrets. But we don't have the tooling in place.
```

Comment:
```suggestion
> Datadog does not have the tooling in place for you to download the Certificate & Provisioning Profile using the [App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi) instead of stored CI secrets.
```

Reply:
Thanks @urseberry for you review 🙏 

I've reverted this change because this readme is for us as internal developers, the note is for future reference if we ever want to build the tooling.